Src/EB/AMReX_EB2_3D_C.cpp

```diff
@@ -121,7 +124,7 @@ void set_eb_data (const int i, const int j, const int k,
         return;
     }
 
-    Real bainv = 1.0_rt/barea(i,j,k);
+    Real bainv = bareascaling*bareascaling/apnorm;
```

Comment:
This is the only place bareascaling is used. barescaling is computed  with std::sqrt(..). So maybe sqrt is unnecessary.